app/config.py

```diff
@@ -72,35 +89,17 @@ def _zone_from_USB(self):
                 self.zone = str(i)
                 return
 
-    def _get_team_specifics(self):
+    def _get_team_image(self):
```

Comment:
This is a better name than `_get_team_specifics` maybe `_get_team_image_path` would be an even better name?